ecommerce/shopper_experience/payment_acquirer.rst

```diff
@@ -1,77 +1,82 @@
-===================================================
-How to manage orders paid with payment acquirers
-===================================================
+================================================
+How To Manage Orders Paid With Payment Acquirers
+================================================
 
-Odoo confirms orders automatically as soon as the payment is authorized 
+Odoo automatically confirms orders as soon as the payment is authorized 
 by a payment acquirer. This triggers the delivery.
 If you invoice based on ordered quantities,
 you are also requested to invoice the order.
 
 
-What are the payment status
-===========================
-At anytime, the salesman can check the transaction status from the order.
+What Are The Different Payment Statuses
+=======================================
+At anytime, the user can check the transaction status from the order, by selecting
+*Payment Transactions* under the *Configuration* menu.
 
-.. image:: media/payment_transaction.png
+.. image:: media/payment-transactions.png
```

Comment:
What happened to the previous image file? If it's no longer used it should be removed.

Reply:
And yet, in this particular case, it shouldn't be removed. Instead, it should be renamed to `payment-transactions.png`.
This is because git doesn't behave like a simple system folder. Instead of just holding the most recent record of the repository content, it remembers *everything*. So, renaming a file actually adds load (equal to the weight of the image file) on the repository. If we rename image files to often, we'll end-up with a multi-GB repository that will take ages to download.